content/help/ops/misc/index.md

```diff
@@ -150,7 +150,7 @@ your Kubernetes services need to be changed slightly.
 
 Kubernetes services must adhere to certain restrictions in order to take advantage of
 Istio's L7 routing features.
-Refer to the [sidecar injection documentation](/docs/setup/kubernetes/sidecar-injection/#pod-spec-requirements)
+Refer to the [Spec requirements](/docs/setup/kubernetes/spec-requirements)
```

Comment:
In this PR, something this link is "Requirements for Pods and Services", "Spec requirements", and "specification requirements". We should be consistent, I think the "Requirements for Pods and Services" is the better alternative.